Project.toml

```diff
@@ -2,15 +2,15 @@ name = "CodecBzip2"
 uuid = "523fee87-0ab8-5b00-afb7-3ecf72e48cfd"
 license = "MIT"
 authors = ["Kenta Sato <bicycle1885@gmail.com>"]
-version = "0.7.2"
+version = "0.8.0"
```

Comment:
Why a breaking change?

Reply:
Just to be on the safe side, since I am not intimately familiar with the Bzip2 api, and it leaves room to update old releases should it be necessary.

Reply:
I see that this creates CompatHelper noise. I'll be more careful.

Reply:
No problem. I guess both options were reasonable to take.